librb/include/rb_memory.h

```diff
@@ -80,26 +80,29 @@ rb_strdup(const char *x)
 	return (ret);
 }
 
-
+// overrule libc's prototype and ignore the cast warning, allowing proper const
```

Comment:
To understand the goal here, it is to enable destruction of objects that should otherwise be `const`, right?